src/cascadia/TerminalCore/Terminal.cpp

```diff
@@ -420,6 +423,33 @@ std::wstring Terminal::GetHyperlinkAtPosition(const COORD position)
         auto uri = _buffer->GetHyperlinkUriFromId(attr.GetHyperlinkId());
         return uri;
     }
+    // also look through our known pattern locations in our pattern interval tree

```

Comment:
For my own clarification, if someone does something like `"^[]8;;https://foo.com^[\https:bar.com^[]8;;^[\"`, and they click it, they're still going to go to foo.com, right?

Reply:
Yes, the way I've implemented it for now is that the OSC 8 one takes priority. However, that is up for discussion - if people think it should be the other way around its an easy fix.

Reply:
Put me down as a vote for the current behavior. I'm thinking of websites where the _real_ URL is something like https://foo.com/some/deep/link#to-a-header, but they just print an anchor tag with "foo.com"